### PR TITLE
implementing new design for faq page

### PR DESCRIPTION
**Pull request by s2sharpit**
## Description 📝
Implementing new Design.

Closes: #480 

## Changes 💬

- Implementing new design to faq page.

### CODE REVIEW

File: package-lock.json
Comment (PiyushKalyanpy) [inline package-lock.json:1]: remove this file

File: src/pages/static/faq.tsx [hunk 1]
```diff
@@ -1,9 +1,107 @@
...
+    <main className="w-screen overflow-hidden lg:block h-fit gap-y-10 dark:bg-neutral-950">
+      <div className="fixed z-40 w-full transition">
+        <LandingNav />
+      </div>
+      <main className="px-6 py-12 md:px-12 max-w-7xl mx-auto grid gap-20">
+        <section className="mt-20 text-center grid gap-3 place-items-center text-zinc-900">
+          <h1 className="text-4xl sm:text-5xl md:text-6xl font-extrabold max-w-4xl">
+            Gyanaguru’s Frequently Asked Question
+          </h1>
+          <p className="text-lg font-medium">
```
Comment (PiyushKalyanpy) [inline src/pages/static/faq.tsx:23]: add more margin : top and button

File: src/pages/static/faq.tsx [hunk 2]
```diff
@@ -1,9 +1,107 @@
...
+  );
+};
+
+
+function FaqSection () {
+  return (
+    <div className="space-y-20">
+      <form className="grid gap-4 max-w-xs mx-auto border-b border-zinc-300">
+        <div className="flex items-center gap-2 rounded p-1">
+          <span className="material-icons">search</span>
```
Comment (PiyushKalyanpy) [inline src/pages/static/faq.tsx:41]: use cursor-pointer

File: src/pages/static/faq.tsx [hunk 3]
```diff
@@ -1,9 +1,107 @@
...
+        </h3>
+        <p className="text-zinc-200 mt-1">
+          We're here to help! Reach out to us for any further inquiries or
+          assistance.
+        </p>
+        <div className="flex gap-4 mt-6">
+          <button className="bg-zinc-800 px-4 py-2 rounded-xl">
+            Learn More
+          </button>
+          <button className="border px-4 py-2 rounded-xl">Contact Us</button>
```
Comment (PiyushKalyanpy) [inline src/pages/static/faq.tsx:93]: on click go to home page contact us form

File: src/pages/static/faq.tsx [hunk 4]
```diff
@@ -1,9 +1,107 @@
...
+        <h3 className="text-zinc-100 text-3xl font-semibold">
+          Still have questions?
+        </h3>
+        <p className="text-zinc-200 mt-1">
+          We're here to help! Reach out to us for any further inquiries or
+          assistance.
+        </p>
+        <div className="flex gap-4 mt-6">
+          <button className="bg-zinc-800 px-4 py-2 rounded-xl">
+            Learn More
```
Comment (PiyushKalyanpy) [inline src/pages/static/faq.tsx:91]: on click go to about us page for now

File: src/pages/static/faq.tsx [hunk 5]
```diff
@@ -1,9 +1,107 @@
...
+          <button className="border px-4 py-2 rounded-xl">Contact Us</button>
+        </div>
+      </div>
+      <div className="w-full lg:w-[50%] self-end mt-4 lg:mt-0 flex justify-end">
+        <Image
+          className="max-h-full max-lg:w-full h-auto lg:max-w-sm"
+          src="/images/faqsection.png"
+          width={800}
+          height={600}
+          alt=""
```
Comment (PiyushKalyanpy) [inline src/pages/static/faq.tsx:102]: add alt text - screenshot of gyanaguru website

File: src/pages/static/faq.tsx [hunk 6]
```diff
@@ -1,9 +1,107 @@
...
     </div>
   );
 }
 
-export default FAQ;
\ No newline at end of file
+function StillHaveQuestions() {
+  return (
+    <section className="bg-zinc-900 text-zinc-200 px-8 lg:px-16 pt-10 rounded-3xl flex flex-wrap lg:flex-nowrap justify-between">
+      <div className="pb-8 w-full lg:w-[50%]">
```
Comment (PiyushKalyanpy) [inline src/pages/static/faq.tsx:81]: remove all the hardcoded values and replace them by tailwind classes 
like  : `w-1/2` in this case

File: src/pages/static/faq.tsx (hunk 2 above)
Comment (PiyushKalyanpy) [inline src/pages/static/faq.tsx:41]: in all material icons, replace `material-icons` class with `material-symbols-outlined`

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

File: src/pages/static/faq.tsx (hunk 1 above)
Comment (s2sharpit) [reply src/pages/static/faq.tsx:23]: Hey @PiyushKalyanpy,
Is it good now?

**Comment by s2sharpit** [comment]
Hey @PiyushKalyanpy,
I have made the changes. Please review it and if needed some more changes let me know.

File: package-lock.json
Comment (PiyushKalyanpy) [inline package-lock.json:1]: please remove this file

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]

**Pull request merged**